app/src/renderer/vuex/modules/config.js

```diff
@@ -5,6 +5,8 @@ export default ({ commit, basecoin }) => {
     activeMenu: '',
     desktop: false,
     devMode: process.env.PREVIEW !== undefined ? JSON.parse(process.env.PREVIEW) : process.env.NODE_ENV === 'development',
+    // TODO: change to atom
+    bondingDenom: 'fermion',
```

Comment:
can we check which network we're using to determine this instead of using a TODO?

Reply:
That would be ideal, but I'm not sure of a good heuristic for checking if a network is a testnet (and I'm not sure what the mainnet will be called in the code). We can do this once we have our first mainnet and need the bonding denom to be 'atom'